tutorindigo/plugin.py

```diff
@@ -117,6 +117,21 @@ def _override_openedx_docker_image(
             "mfe-dockerfile-post-npm-install-authn",
             """
 RUN npm install '@edx/brand@npm:@edly-io/indigo-brand-openedx@^1.0.0'
+""",
+        ),
+        (
+            "mfe-dockerfile-post-npm-install-discussions",
```

Comment:
Why are these components installed only in the discussions MFE? Same question for the learner dashboard below.

Reply:
In this release, Discussion MFE and learner-dashboard styles has been updated. That's why brand-openedx is added for styling and header component and footer component are added for header and footer. Moreover, few fixes have been done for learning and authn mfe (fixes are related to styling).

Reply:
Can you please add a note as a comment? Otherwise I'm afraid our future selves will forget why these plugins are added only for a few plugins.